pkgs/development/python-modules/pyp/default.nix

```diff
@@ -1,56 +1,68 @@
 { lib
+, astunparse
+, bc
 , buildPythonPackage
 , fetchFromGitHub
-, pytestCheckHook
-, coreutils
-, pythonOlder
-, astunparse
 , flit-core
 , jq
-, bc
+, pytestCheckHook
+, pythonOlder
 }:
 
-buildPythonPackage rec {
-  pname = "pyp";
-  version = "1.2.0";
-  format = "pyproject";
+let
+  finalAttrs = {
```

Comment:
There is really no point in this custom finalAttrs construction. We should work torwards enabling it with buildPythonPackage instead.

Reply:
Yes. I wanna keep this anyway - the finalAttrs migration is going too slow.